app/renderer/js/notification/helpers.ts

```diff
@@ -82,7 +82,7 @@ const webContentsId = webContents.id;
 export function focusCurrentServer(): void {
 	// TODO: TypeScript: currentWindow of type BrowserWindow doesn't
 	// have a .send() property per typescript.
-	(currentWindow as any).send('focus-webview-with-id', webContentsId);
+	currentWindow.webContents.send('focus-webview-with-id', webContentsId);
```

Comment:
Why webcontents is added ? This is wrong.

Reply:
BrowserWindow as per electronjs docs, doesn't have any .send() property. It is a property used as BrowserWindow.webContents.send().
That's why I did this.

Reply:
Test the functionality. If it works this way, then it is fine.

Reply:
Tested. Works Fine.